### PR TITLE
r-deldir

### PR DESCRIPTION
**Pull request by jsignell**

### CODE REVIEW

File: recipes/r-deldir/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
```
Comment (isuruf) [inline recipes/r-deldir/meta.yaml]: You need `libgcc` for unixes

**Comment by ocefpaf** [comment]
@jsignell it seems I botched this one when adding the `libgcc` dep now GitHub does not allow me to resolve the conflicts. Can you do that locally? If not I can pick from your fork and fix this manually.

**Comment by bgruening** [comment]
r-deldir was added already https://anaconda.org/conda-forge/r-deldir

**Comment by ocefpaf** [comment]
Thanks @bgruening and @jsignell. Closing this.

**Pull request closed without merging**